package.json

```diff
@@ -108,11 +106,12 @@
     "typescript": "^3.9.7",
     "url-loader": "^4.1.0",
     "webpack": "^4.44.1",
```

Comment:
Default webpack version should be `5`?

Reply:
@ylemkimon I think we should update dev deps to latest too, but let's do it in separate PR

Reply:
I think we should enable renovate on the `v4` branch.

Reply:
After merge it in master, it will be automatically, but to avoid spamming and weird problems with CI and tests better to do it in separate PR (I think)

Reply:
yep, maybe need improve some tests

Reply:
@evilebottnawi Using webpack 5 as the dev dependency would mean we're compiling client sources with webpack 5. Wouldn't this cause problem with old browser compatibility?

Reply:
Should be no, but we should set `target: ['web', 'es5']`